pkg/util/homedir/homedir.go

```diff
@@ -23,7 +23,7 @@ import (
 
 // HomeDir returns the home directory for the current user
 func HomeDir() string {
-	if runtime.GOOS == "windows" {
+	if runtime.GOOS == "windows"; len(os.Getenv("HOME")) <= 0) {
```

Comment:
Leave this line as is and do, right below inside the windows condition:

```
if home := os.Getenv("HOME"); len(home) > 0 {
  if _, err := os.Stat(home); err == nil {
    return home
  }
}
```

on Windows we always need a `os.Stat` to make sure it's good to use.